configure.py

```diff
@@ -1102,6 +1102,7 @@ def find_ninja():
                 'service/qos/qos_table.cc',
                 'service/qos/service_level_controller.cc',
                 'service/qos/standard_service_level_distributed_data_accessor.cc',
+                'service/qos/raft_service_level_distributed_data_accessor.cc',
                 'streaming/stream_task.cc',
```

Comment:
Missing CMakeLists.txt

Reply:
@avikivity could you explain what am I missing?